docs/explanation/package-model.rst

```diff
@@ -1,9 +1,196 @@
+
+- ``ubiquity_23.10.2.dsc``:  The *Debian Source Control* file of the *source package*.
+- ``ubiquity_23.10.2.tar.xz``: The tarball containing the :term:`Source Code` of the project.
+
+.. _SourcePackageFormat_1.0:
+
+Format: ``1.0``
+^^^^^^^^^^^^^^^
+
+The original *source package* format. Nowadays, this format is rarely used.
+
+A native *source package* in this format consists of a single ``.tar.gz``
+file containing the :term:`Source`.
+
+A non-native *source package* in this format consists of a ``.orig.tar.gz`` file
+(containing the :term:`Upstream` :term:`Source`) associated with a ``.diff.gz``
+file (the :term:`Patch` containing :term:`Debian` packaging modifications).
+Optionally, the original tarball can be accompanied by a detached :term:`Upstream`
+signature ``.orig.tar.gz.asc``. 
+
+.. note::
+  
+  This format does not specify a :term:`Patch` system, which makes it harder for
+  :term:`Maintainers <Maintainer>` to track modifications. There were multiple approaches
+  to how packages tracked :term:`Patches <Patch>`. Therefore, the *source packages* of this
+  format often contained a :file:`debian/README.source` file explaining how to use the 
+  :term:`Patch` system.
+
+``3.0`` Formats Improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some of the improvements that apply to most ``3.0`` formats are:
+
+- Support for additional compression formats: ``bzip2``, ``lzma`` and ``xz``
+- Support for multiple :term:`Upstream` tarballs
+- Supports inclusion of binary files
+- :term:`Debian`-specific changes are no longer stored in a single ``.diff.gz``
+- The :term:`Upstream` tarball does not need to be repacked to strip the :term:`Debian` directory.
+
+Other Formats
+^^^^^^^^^^^^^
+
+The following formats are rarely used, experimental and/or historical.
+You should only choose these if you know what you do.
+
+- ``3.0 (custom)``: Doesn't represent an actual *source package* format but can
+  be used to create *source packages* with arbitrary files.
+- ``3.0 (git)``: An experimental format to package from a :term:`git` repository.
+- ``3.0 (bzr)``: An experimental format to package from a :term:`Bazaar` repository.
+- ``2.0``: The first specification of a new-generation *source package* format.
+  It was never widely adopted and eventually replaced by
+  :ref:`3.0 (quilt) <SourcePackageFormat_3.0quilt>`.
+
+``.changes`` file
+~~~~~~~~~~~~~~~~~
+
+Although technically not part of a *source package* -- every time a *source package* is built,
+a :file:`.changes` file will be created alongside. The :file:`.changes` file contains metadata
+from the *Source Control* file and other information (e.g. the latest changelog entry) about the
+*source package*. :term:`Archive` tools and :term:`Archive Administrators <Archive Admin>` use this
+data to process changes to *source packages* and determine the appropriate action to upload the 
+*source package* to the :term:`Ubuntu Archive`.
+
+.. _BinaryPackages:
+
+Binary Packages
```

Comment:
```suggestion
Binary packages
```

Reply:
I would actually recommend moving this to above the source packages section, since this one is so short and binary packages are referenced elsewhere - I think it's important not to bury this bit at the end of the page